libbpf-tools/javagc.bpf.c

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+/*
+ * Author: Chen Tao
+ * Create: Wed Jun 29 16:00:19 2022
+ */
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+#include <bpf/usdt.bpf.h>
```

Comment:
You don't use `BPF_USDT`, so drop `usdt.bpf.h`.

Reply:
If i remove the header file, it will report "libbpf: usdt: failed to find USDT support BPF maps, did you forget to include bpf/usdt.bpf.h?", and no tracing event report.

Reply:
OK, that's fine.